pom.xml

```diff
@@ -0,0 +1,239 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
Don't merge. It looks like the scripts missed the grpc and proto libs because of a naming diff.